language/core.german.lang

```diff
@@ -358,7 +358,7 @@ Baue eine Telnetverbindung zu dem Bot auf und gib 'NEW' als Deinen Nickname ein.
 0xb5d,Breche die Verbindung ab zu
 0xb5e,Relais kaputt
 0xb5f,Ping reagiert nicht
-0xb60,Unleafiges Verhalten
+0xb60,Unleidiges Verhalten
```

Comment:
The intention is to say "not a leaf bot" which is a bot that's not a hub in a botnet. I don't think there is a good translation, but "unleidig" doesn't improve it, "unleafig" was better.

```suggestion
0xb60,Unleafiges Verhalten
```